storybook/src/admin/table/TableStackEditDialogHooks.tsx

```diff
@@ -30,7 +29,7 @@ interface IExampleRow {
 }
 
 interface IEditFormProps {
-    row: IExampleRow;
+    row?: IExampleRow;
```

Comment:
```suggestion
    row: IExampleRow;
```

I don't think this is necessary since there's an early return in [L122-124](https://github.com/vivid-planet/comet/pull/2034/files#diff-5a9ecbf025c63593ee1ce81767357c78122af20fe348fd7adcba77d4023f818cR122-R124)